File: Notas de aula 5/Apresentação/for.js
oqueRepet = prompt("O quê deseja repetir?")
vezesRepet = prompt("Digite quantas vezes deseja repetir.")
if (Number.isFinite(vezesRepet)) {
    alert("Precisa ser um número.")
} else {
    for (i = 0; i < vezesRepet; i++) {
        alert(oqueRepet);
    }
}

File: Notas de aula 5/Apresentação/for.cpp
#include <iostream>
#include <cmath>
int main()
{
    using namespace std;
    string  oqueRepet;
    float vezesRepet, i;
    cout << "O quê deseja repetir?\n";
    cin >> oqueRepet;
    cout << "Quantas vezes deseja repetir?\n";
    cin >> vezesRepet;
    if (isfinite(vezesRepet)) {
    for (i=0; i<vezesRepet; i++) {
        cout << oqueRepet << endl;
    }
    } else {
        cout << "Especifique o número!!\n";
    }
    return 0;
}
